gradle.properties

```diff
@@ -0,0 +1,12 @@
+org.gradle.caching=true
+group=io.ballerina.stdlib
+version=0.1.0-SNAPSHOT

```

Comment:
version?

Reply:
Fixed in [0df993b](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/134/commits/0df993b9d0ae1de867b6aa56ae639eecc72b24b8)